### DISCUSSION TITLE
size

### DISCUSSION THREAD

**Discussion by jinyongp** [remeda/remeda#947] [Suggest a Function]
Hi there,

I noticed that the `length` function is provided as an alternative to Lodash’s `size`, but it only works with iterable objects. It’d be great to have a `size` function that behaves like Lodash’s, handling different types of objects like Object, Set, and Map.

If this sounds good, I’d be happy to work on it and send a PR.

Also, I saw some discussion about `countBy`. what do you think about adding a `sizeBy` function too? I think it could be pretty handy.

Thanks for all your hard work!

**Comment by eranhirsch** [comment]
I think size makes sense and would be a good addition to the library. I don't understand how sizeBy would work, can you give an example?

**Comment by jinyongp** [comment]
Oh, I just realized I misunderstood how `countBy` works. Now that I think about it, `sizeBy` doesn't seem necessary.
